src/dsl/verifier.ts

```diff
@@ -158,6 +165,65 @@ export class Verifier {
     }
   }
 
+  private createRequestTracer(): express.RequestHandler {
+    return (req, _, next) => {
+      logger.trace(
+        "incoming request",
+        pickBy(
```

Comment:
This doesn't read clearly to me (I had to go and read the definition of `pickBy`). Perhaps this could be extracted to a separate function, called something like `removeEmptyProperties`?

Reply:
Can do.